storybookjs/storybook
# Add preview type and update CLI template

Closes #20950

## What I did

* export Preview from all renderers
* Update CLI templates

I didn't update the documentation yet, or added a runtime warning for old style preview.js files. Maybe we can outscope that for now?

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Check the generated preview! 
-->

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
